src/BaseTestNext.jl

```diff
@@ -315,7 +315,10 @@ record(ts::DefaultTestSet, t::Pass) = (push!(ts.results, t); t)
 function record(ts::DefaultTestSet, t::Union{Fail,Error})
     print_with_color(:white, ts.description, ": ")
     print(t)
-    Base.show_backtrace(STDOUT, backtrace())
+    # printing Errors already prints the backtrace of the error
+    if typeof(t) != Error
```

Comment:
Good point. I just fixed (as well as another place with the same issue). I force-pushed it to my branch but it doesn't look like GH is picking it up yet. Hopefully it will shortly.